integration_tests/mockApis/probationOffenderSearchApi.ts

```diff
@@ -0,0 +1,24 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+export default {
+  stubMatchPrisoners: (prisonerNumbers: string[] = []): SuperAgentRequest =>
+    stubFor({
+      request: {
+        method: 'POST',
+        urlPattern: `nomsNumbers`,
```

Comment:
if you had issues with this being `/nomsNumbers`, change not to use `urlPattern`, just `url`

i lost so much time with `urlPattern` in the past